FIXES

```diff
@@ -202,7 +202,7 @@ January 9, 2020:
 	mere warnings. Thanks to Martijn Dekker <martijn@inlv.org>.
 
 January 5, 2020:
-	Fix a bug in the concatentation of two string constants into
+	Fix a bug in the concatenation of two string constants into
```

Comment:
Some projects don't like changing changelogs. I'm happy to drop any changes...

Reply:
i have no idea what is going on here. I don't mind spelling corrections, but this is quite something. an list in an issue instead of this long chain of discussions and 16 commits would have sufficed.

Reply:
> a simple list as an issue instead of this long chain of discussions and 16 commits would have sufficed. in fact, I will ask you to drop this and do just that.

The commits can be squashed on merge. No reason to change anything here.